src/main/resources/assets/api-docs.yaml

```diff
@@ -108,6 +108,8 @@ paths:
           description: Deprecated endpoint. Use  `GET /api/tdr/{identifier}` instead.
   /api/daa/{daaId}:
     $ref: 'paths/daaByDaaId.yaml'
+  /api/daa/{daaId}/{userId}:
+    $ref: 'paths/daaForUserId.yaml'
```

Comment:
This is duplicated on line 117 - we only need one. The swagger page will show these kinds of issues pretty quickly ... 
<img width="1351" alt="Screenshot 2024-04-02 at 8 26 29 AM" src="https://github.com/DataBiosphere/consent/assets/116679/e1354675-4c41-4bd8-b3c9-6be4f7c0b9f9">